conf/messages

```diff
@@ -258,6 +263,31 @@ paye.ats.treasury_spending.para2=All figures are rounded to the nearest pound.
 paye.ats.treasury_spending.para3=The figures in the table above are intended as an illustration of how taxes are spent and not as a direct link between your Income Tax, National Insurance contributions and any specific expenditure.
 paye.ats.treasury_spending.para4=Spending information is published by HM Treasury.
 
+# View ATS - Summary (PAYE)
+#============================================================
+paye.ats.summary.html.title=Your income and taxes
+paye.ats.summary.title=Your income and taxes
+paye.ats.summary.lede=This is for your information only. You do not need to contact us as this is not a bill or demand for payment.
+paye.ats.summary.total_income.title=Taxable income
+paye.ats.summary.total_income.link=Your taxable income
+paye.ats.summary.total_income.description=This is your total taxable income for the year.
+paye.ats.summary.tax_free.title=Tax-free amount
+paye.ats.summary.tax_free.link=Your tax-free amount
+paye.ats.summary.tax_free.description=This is the amount you received without paying tax.
+paye.ats.summary.tax_and_nics.title=Income Tax and National Insurance
+paye.ats.summary.tax_and_nics.link=Your Income Tax and National Insurance contributions
+paye.ats.summary.tax_and_nics.description=This is {0}% of your taxable income. For every £1 of income, you paid {0}p in Income Tax and National Insurance contributions.
```

Comment:
typo with {0}p

Reply:
This is "p" as in "pence" so I believe is correct - the value as a percentage and as a proportion of £1 in pence is the same value, {0}, being used in two places.